client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+
+class PluginAutomatedTransfer extends Component {
+
+	static propTypes = {
+		translate: PropTypes.func,
+		plugin: PropTypes.object,
+	}
+
+	static defaultProps = {
+		plugin: {
+			name: 'Plugin',
+		},
+	}
+
+	state = {
+		transferStep: 'start',
+	}
+
+	setNoticeText = ( pluginName = '' ) => {
+		const { translate } = this.props;
+		const { transferStep } = this.state;
+
+		switch ( transferStep ) {
+			case 'start': return translate( 'Installing %s…', {
+				args: pluginName,
+				comment: 'Installing plugin…',
+			} );
+			case 'setup': return translate( 'Now configuring your site. This may take a few minutes.' );
+			case 'leaving': return translate( "Don't leave quite yet! Just a bit longer." );
+			case 'conflicts': return translate( 'Sorry, we found some conflicts to fix before proceding.' );
+			case 'complete': return translate( 'Successfully installed %s!', {
+				args: pluginName,
+				context: 'Successfully installed plugin!',
```

Comment:
this isn't really a context; it's a comment. let's change it around otherwise it's going to make the job more difficult and expensive for our translators. in this case, I don't think we'd need a context at all because this string doesn't seem to have multiple meanings.